scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -361,6 +361,23 @@ public Boolean call() throws Exception {
         }
     }
 
+    public boolean changeStartAt(final JobId jobId, final String startAt) {
+        try {
+            if (status.isShuttingDown()) {
+                return false;
+            }
+            return infrastructure.getClientOperationsThreadPool().submit(new Callable<Boolean>() {
+                @Override
+                public Boolean call() throws Exception {
+                    return jobs.updateStartAt(jobId, startAt);
+                }
+
+            }).get();
```

Comment:
Nice you wait with the .get() until the startAt is changed to prevent race conditions. 
In the unlikely case the REST call timeouts while waiting for a thread, the startAT will still be set.